sklearn/neighbors/graph.py

```diff
@@ -67,8 +72,7 @@ def kneighbors_graph(X, n_neighbors, mode='connectivity', metric='minkowski',
 
     include_self : bool, default=False.
         Whether or not to mark each sample as the first nearest neighbor to
-        itself. If `None`, then True is used for mode='connectivity' and False
```

Comment:
Is this no longer applicable? If so, why do you implement it in the Transformer?

Reply:
First I removed the doc since it was not implemented in `kneighbors_graph`.
Then I ended up implementing it in the transformers since it is a better default for exact correspondance with internal NN in most classes.
I think I will re-implement it in the function as well.